types/clevertap-web-sdk/test/clevertap-global-tests.ts

```diff
@@ -0,0 +1,92 @@
+/// <reference types="../index" />
```

Comment:
```suggestion
```
either use proper import (`import clevertap = require('.....');`, or remove reference in tests. If global script version used, it does not require explicit reference in tests

Reply:
Wasn't aware of this. Thanks 👍